map/page.js

```diff
@@ -18,11 +18,76 @@ const Latitude = "Latitude";
 const Geocode = 'Geocode';
 // Optional - but required for geocoding. Field with address to find (might be formula)
 const Address = 'Address';
-// Optional - but 
+// Optional - but
```

Comment:
Do you have a guess for what this comment was trying to say? but ... ditto?

Reply:
Something like "Optional - but useful for geocoding. Blank field which map uses to store last geocoded Address. Enables map widget to automatically update the geocoding if Address is changed"

These comments look systematic, do these get parsed and sent to grist UI somewhere? If so my description might be too long.